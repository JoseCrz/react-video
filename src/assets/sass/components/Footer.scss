.Footer {
  display: flex;
  align-items: center;
  height: 100px;
  width: 100%;

  &__link {
    color: white;
    cursor: pointer;
    font-size: 14px;
    margin-left: 24px;
    text-decoration: none;

    &:hover {
      text-decoration: underline;
    }
  }
}