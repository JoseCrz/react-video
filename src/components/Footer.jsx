import React from 'react'

import '../assets/sass/components/Footer.scss'

const Footer = () => {
  return (
    <footer className="Footer">
      <a className="Footer__link" href="/">Terms & Conditions</a>
      <a className="Footer__link" href="/">Privacy</a>
      <a className="Footer__link" href="/">Help Center</a>
    </footer>
  )
}

export default Footer