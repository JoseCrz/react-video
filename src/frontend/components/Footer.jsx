import React from 'react'
import { Link } from 'react-router-dom'

import '../assets/sass/components/Footer.scss'

const Footer = () => {
  return (
    <footer className="Footer">
      <Link to="/" className="Footer__link">Terms & Conditions</Link>
      <Link to="/" className="Footer__link">Privacy</Link>
      <Link to="/" className="Footer__link">Help Center</Link>
    </footer>
  )
}

export default Footer