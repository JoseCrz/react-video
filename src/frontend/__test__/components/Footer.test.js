import React from 'react'
import { mount } from 'enzyme'
import Footer from '../../components/Footer'
import { create } from 'react-test-renderer'

describe('<Footer /> Tests', () => {
  const footer = mount(<Footer />)

  test('Component Rendering', () => {
    expect(footer.length).toEqual(1)
  })

  test('Footer has 3 anchors', () => {
    expect(footer.find('a')).toHaveLength(3)
  })

  test('Footer Snapshot', () => {
    const footer = create(<Footer />)
    expect(footer.toJSON()).toMatchSnapshot()
  })

})
