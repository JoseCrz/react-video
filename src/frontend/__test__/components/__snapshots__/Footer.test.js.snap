// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Tests Footer Snapshot 1`] = `
<footer
  className="Footer"
>
  <a
    className="Footer__link"
    to="/"
  >
    Terms & Conditions
  </a>
  <a
    className="Footer__link"
    to="/"
  >
    Privacy
  </a>
  <a
    className="Footer__link"
    to="/"
  >
    Help Center
  </a>
</footer>
`;
